src/prefect/new_task_runners.py

```diff
@@ -64,6 +73,27 @@ def submit(
         """
         ...
 
+    @abc.abstractmethod
+    def map(
+        self,
+        task: "Task",
+        parameters: Dict[str, Any],
+        wait_for: Iterable[PrefectFuture],
+    ) -> Iterable[PrefectFuture]:
```

Comment:
I love that you moved this "down" to the `TaskRunner` interface, it will be so nice to integrate with other platforms and use their native features without dragging complexity "up" into Prefect :bow: